src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml

```diff
@@ -46,17 +46,17 @@
         </Grid.ColumnDefinitions>
         <DataGrid Grid.Column="0" 
                   SelectedItem="{Binding SelectedIssue, Mode=TwoWay}"
-                  ItemsSource="{Binding Issues}"
+                  ItemsSource="{Binding IssuesView}"
```

Comment:
The control now binds explicitly to the exposed `ICollectionView`, rather than implicitly creating one behind the scenes.